### PR TITLE
(Feature) e2e tests for Wizard

### PR DESCRIPTION
**Pull request by vbaranov**
Closes https://github.com/poanetwork/token-wizard/issues/509

**Problem**: we have no integrated e2e tests for Token Wizard
**Solution**: integrate `token-wizard-test-automation` as a submodule

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,16 @@ install:
 
 before_script:
   - npm run installWeb3
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - npm i -g npm@^3
+  - sleep 3
```
Comment (fvictorio) [inline .travis.yml]: Why is this necessary?

Comment (vbaranov) [reply .travis.yml]: @dennis00010011b could you clarify why it is necessary?

Comment (dennis00010011b) [reply .travis.yml]: Xvfb is needed for performs graphical operations in virtual memory. e2e script uses Metamask extension but  headless Chrome don't support extension. It is why Xvfb used

Comment (fvictorio) [reply .travis.yml]: I was asking about the `sleep 3` line.

Comment (dennis00010011b) [reply .travis.yml]: It is recommended code from https://docs.travis-ci.com/user/gui-and-headless-browsers/
``` sleep 3 ```  give xvfb some time to start

Comment (vbaranov) [reply .travis.yml]: @dennis00010011b did you try other setups from recommended in the link above? if yes, do they work? just curious.

**Comment by vbaranov** [comment]
@fvictorio @fernandomg let's do not merge any PR until we will not add e2e tests.

File: .travis.yml (hunk 1 above)
Comment (dennis00010011b) [reply .travis.yml]: Not too much. I just tried to play with screen resolution but it wasn't helpfull

Comment (dennis00010011b) [reply .travis.yml]: Line 12 ```- npm i -g npm@^3``` should be excluded

**Comment by vbaranov** [comment]
@fernandomg @fvictorio is there anything stops us from merging?

**Comment by fernandomg** [comment]
@vbaranov I'm running it locally, but I'm getting the following

```
[2018-03-07T14:05:08.235Z]  Test PASSED.'Owner can  finalize (after all tokens were sold)  
    ✓ Owner can  finalize (after all tokens were sold) (75902ms)
    ✓ Investors receive right amount of tokens after finalization)


  14 passing (13m)
  2 failing

  1) POA token-wizard. Test suite #1
       Investor can buy  amount = minCap:

      AssertionError [ERR_ASSERTION]: Test FAILED.Investor can NOT buy minCap
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (test/testSuite1.js:158:16)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
  From: Task: POA token-wizard. Test suite #1 Investor can buy  amount = minCap
      at Context.ret (node_modules/selenium-webdriver/testing/index.js:182:10)

  2) POA token-wizard. Test suite #1
       Investor can buy less than minCap after first transaction:

      AssertionError [ERR_ASSERTION]: Test FAILED. Investor can NOT buy less than minCap after first transaction
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (test/testSuite1.js:173:16)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
  From: Task: POA token-wizard. Test suite #1 Investor can buy less than minCap after first transaction
      at Context.ret (node_modules/selenium-webdriver/testing/index.js:182:10)
```

**Comment by vbaranov** [comment]
@fernandomg please try to restart tests. You can see it worked with this PR. Anyway, @dennis00010011b will try to increase stability in the next releases of test-automation script.

**Comment by dennis00010011b** [comment]
@fernandomg There is should be result.log file(see path ./results/result_date_time/) Could you send it me?

**Comment by fernandomg** [comment]
> There is should be result.log file(see path ./results/result_date_time/) Could you send it me?

Sorry, I moved to another issue and cleaned up the workspace. Now I worked. Thanks

**Comment by fvictorio** [comment]
@vbaranov I think this can be merged now.
